README.md

```diff
@@ -1,84 +1,107 @@
-6. Delete these instructions and everything up to the _Project Title_ from the README.
-7. Write some great software and tell people about it.
+# HMDA Platform
 
-> Keep the README fresh! It's the first thing people see and will make the initial impression.
+## This project is a work in progress
 
-----
+Information contained in this respository should be considered provisional and a work in progress, and not the final implementation for the HMDA Platform, unless otherwise indicated. 
+
+## Introduction
+
+The Home Mortgage Disclosure Act (HMDA) requires many financial institutions to maintain, report, and publicly disclose information about mortgages. HMDA was originally enacted by Congress in 1975 and is implemented by [Regulation C](https://www.gpo.gov/fdsys/pkg/CFR-2012-title12-vol8/xml/CFR-2012-title12-vol8-part1003.xml). The Dodd-Frank Act transferred HMDA rulemaking authority from the Federal Reserve Board to the Consumer Financial Protection Bureau (CFPB) on July 21, 2011.
+
+This regulation provides the public loan data that can be used to assist:
+* in determining whether financial institutions are serving the housing needs of their communities;
+* public officials in distributing public-sector investments so as to attract private investment to areas where it is needed;
+* and in identifying possible discriminatory lending patterns.
+
+This regulation applies to certain financial institutions, including banks, savings associations, credit unions, and other mortgage lending institutions.
+
+## The HMDA Platform
 
-# Project Title
+This repository contains the code for the entirety of the HMDA platform backend. This platform has been designed to accomodate the needs of the HMDA filing process by financial institutions, as well as the data management and publication needs of the HMDA data asset. 
 
-**Description**:  Put a meaningful, short, plain-language description of what
-this project is trying to accomplish and why it matters.
-Describe the problem(s) this project solves.
-Describe how this software can improve the lives of its audience.
+The HMDA Platform is composed of the following modules: 
 
-Other things to include:
+### Parser
 
-  - **Technology stack**: Indicate the technological nature of the software, including primary programming language(s) and whether the software is intended as standalone or as a module in a framework or other ecosystem.
-  - **Status**:  Alpha, Beta, 1.1, etc. It's OK to write a sentence, too. The goal is to let interested people know where this project is at. This is also a good place to link to the [CHANGELOG](CHANGELOG.md).
-  - **Links to production or demo instances**
-  - Describe what sets this apart from related-projects. Linking to another doc or page is OK if this can't be expressed in a sentence or two.
+Component responsible for reading incoming data and making sure that it conforms to the HMDA File Specification
```

Comment:
We use the term "module" in the subsequent sections.  Should we make this consistent?